packages/atlas-service/src/atlas-auth-service.ts

```diff
@@ -4,7 +4,7 @@ import type { AtlasUserInfo } from './util';
 export type ArgsWithSignal<T = Record<string, unknown>> = T & {
   signal?: AbortSignal;
 };
-export type SignInPrompt = 'none' | 'ai-promo-modal';
+export type SignInPrompt = 'none';
```

Comment:
Good callout, it could use an intentional function for each I feel. It's a bit overloaded without benefit at the moment. I'll see what we can do here.